{
  assertEquals(1D,Utility.remainderWithoutOverflow(25D,4D),0D);
  try {
    Utility.remainderWithoutOverflow(5D,0D);
    fail("did not get expected exception");
  }
 catch (  ArithmeticException expected) {
  }
}
